meshnets/modules/encoder.py

```diff
@@ -18,11 +19,13 @@ def __init__(self, node_feats_size, mesh_feats_size, latent_size,
         given the features sizes, latent size and number of MLP layers."""
         super().__init__()
 
-        # TODO(victor) : implement encoders
-        self.node_encoder = Identity(node_feats_size, latent_size,
-                                     num_mlp_layers)
-        self.edge_encoder = Identity(mesh_feats_size, latent_size,
-                                     num_mlp_layers)
+        node_encoder_widths = [node_feats_size
+                              ] + (num_mlp_layers + 1) * [latent_size]
+        self.node_encoder = MLP(node_encoder_widths, layer_norm=True)
+
+        edge_encoder_widths = [mesh_feats_size
```

Comment:
mesh or edges features? Just to understand that this are local features, right?

Reply:
You're right, edge features are probably more correct here, I changed the argument name